metrics/src/main/java/metrics/MetricsServletContextListener.java

```diff
@@ -0,0 +1,21 @@
+package metrics;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.servlets.MetricsServlet;
+
+import javax.servlet.annotation.WebListener;
+
+/**
+ * Annotated context listener for wiring up the metrics servlet.
+ */
+@WebListener
+public class MetricsServletContextListener extends MetricsServlet.ContextListener {
```

Comment:
This needs to implement the abstract non-static getter required by the abstract ContextListener Dropwizard class.